stdlib/LinearAlgebra/src/exceptions.jl

```diff
@@ -29,10 +29,24 @@ function Base.showerror(io::IO, ex::ARPACKException)
     end
 end
 
+"""
+    SingularException
+
+The input matrix has one or more zero-valued eigenvalues, and is not invertible.
```

Comment:
I think this should describe the exception instead, e.g. "Exception thrown when..."

Reply:
👍